massgov/mayflower-patternlab
# Dp 6517 page alerts client side data

## Description

This change provides a jQuery plugin to process alert data.  It provides the option of using a transform method between an endpoint and the render process so that if the endpoint cannot be updated to provide exactly what the component needs, it can be transformed.

## Related Issue / Ticket
- https://jira.state.ma.us/browse/DP-6517

## Steps to Test


1. Visit https://tnathanjames.github.io/mayflower/?p=organisms-ajax-alerts
2. Confirm that the following alert is shown:
![image](https://user-images.githubusercontent.com/5727673/32680199-2ddd2db6-c62f-11e7-8a32-129dfe40c849.png)
3. Visit https://tnathanjames.github.io/mayflower/patterns/05-pages-guide/05-pages-guide.html
4. Confirm that you see a similar alert.

## Additional Notes:

The ability to implement this outside of mayflower needs to be tested.

#### Impacted Areas in Application


#### @TODO


#### Today I learned...
